src/ark/segmentation/ez_seg/merge_masks.py

```diff
@@ -165,3 +177,54 @@ def merge_masks_single(
 
     # Return unmerged cells
     return cell_labels
+
+
+def get_bounding_boxes(object_labels: np.ndarray):
+    """
+    Gets the bounding boxes of labeled images based on object major axis length.
+
+    Args:
+        object_labels (np.ndarray): label array
+    Returns:
+        dict: Dictionary containing labels as keys and bounding box as values
+    """
+    bounding_boxes = {}
+
+    # Get region properties as a DataFrame
+    props = regionprops_table(object_labels, properties=('label', 'bbox'))
+
+    # Convert to DataFrame
+    df = pd.DataFrame(props)
+
+    for _, row in df.iterrows():
+        label_id, min_row, min_col, max_row, max_col = row.values
+        # Return closed interval bounding box
+        bounding_boxes[label_id] = ((min_row, min_col), (max_row-1, max_col-1))
+
+    return bounding_boxes
+
+
+def filter_labels_in_bbox(bounding_box: List, cell_props: pd.DataFrame, expansion_factor: int):
```

Comment:
Nothing to change here, but if you're really feeling like going _fast_, take a look at [Numba accelerated Pandas functions](https://pandas.pydata.org/docs/user_guide/enhancingperf.html#custom-function-examples). It would require some rearranging, but it'll provide more speedups if necessary.

Reply:
looks interesting - will keep as is for now since additional speedup isn't necessary atm.